src/etc/local_stage0.sh

```diff
@@ -49,6 +49,13 @@ if [ -z $TARG_DIR ]; then
     exit 1
 fi
 
+case "$TARG_DIR" in
+--print-rustc-release)
+  # not actually copying to TARG_DIR, just print the local rustc version and exit
+  ${PREFIX}/bin/rustc${BIN_SUF} --version --verbose | sed -ne 's/^release: \(.*\)/\1/gp'
```

Comment:
Not a big deal, but the wildcard is unnecessary.  Just `s/^release: //p` will work.